easybuild/easyblocks/r/rosetta.py

```diff
@@ -249,8 +247,6 @@ def extract_and_copy(dirname_tmpl, optional=False, symlinks=False):
         # (extract and) copy database, docs, demos (incl tutorials) and biotools (if it's there)
         if os.path.exists(os.path.join(self.cfg['start_dir'], 'main', 'database')):
             extract_and_copy(os.path.join('main', 'database') + '%s')
-        else:
-            extract_and_copy('rosetta_database%s')
```

Comment:
I expect this is needed for older versions. If this is no longer useful for 3.13, then we should change this to a version check, so:
```
        elif LooseVersion(self.version) < LooseVersion("3.13"):
            extract_and_copy('rosetta_database%s')

Reply:
I think I uploaded a wrong version of the file. I have just updated the right file on my git repo [73dc29d](https://github.com/alikerr/easybuild-easyblocks/commit/73dc29d19e29cb8c94f00cddded0ecc9c44c1d5b)